tests/test_tester_reification.py

```diff
@@ -0,0 +1,54 @@
+import unittest
+
+import clingo
+from clingox.program import Program, ProgramObserver, Remapping
+
```

Comment:
Let's ignore this code for the moment on this branch. I'll create an independent lute/test_tester_reification branch for it.